document-classification/content.md

```diff
@@ -295,7 +295,8 @@ import mlflow
 import mlflow.sklearn
 import mflux_ai
 
-mflux_ai.set_env_vars("Insert your key here")
+# Note: in the following line, insert the project token shown on your dashboard page.
+mflux_ai.set_env_vars("your_project_token_goes_here")
```

Comment:
I have changed the API from `mflux_ai.set_env_vars(...)` to `mflux_ai.init(...)`. The old one will still work, but let's use the new one in the tutorials going forward :)